api/accounts/accounts_test.go

```diff
@@ -527,7 +527,7 @@ func httpPost(t *testing.T, url string, body interface{}) ([]byte, int) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	res, err := http.Post(url, "application/x-www-form-urlencoded", bytes.NewReader(data))
+	res, err := http.Post(url, "application/x-www-form-urlencoded", bytes.NewReader(data)) // nolint:gosec
```

Comment:
Could optionally disable gosec for `*_test.go`

Reply:
True, I think typically it's preferable to have the need to be aware of the why/how for disabling the linter in such particular cases.